### PR TITLE
Start 1.9 release cycle

### PR DESCRIPTION
**Pull request by imphil**

### CODE REVIEW

File: documentation/source/release_notes.rst [hunk 1]
```diff
@@ -11,6 +11,40 @@ All releases are available from the `GitHub Releases Page <https://github.com/co
...
+- Fixed :class:`RecursionError` caused by certain corner cases in the scheduler. (:pr:`3267`)
+- Fixed cleanup in scheduler which caused sporadic warning messages and bugs in some corner cases. (:pr:`3270`)
+- Fix "use after free" bug in VHPI implementation causing Riviera to fail to discover some simulation objects. (:pr:`3307`)
+
+
+Changes
+-------
+
+- Removed ``level`` arg from ``_sim_event`` function in the :envvar:`PYGPI_ENTRY_POINT` interface. This function can only indicate a request to shutdown from the simulator or GPI. (:pr:`3066`)
+- Moved :class:`cocotb.task.Task` and friends to ``cocotb.task`` module to alleviate internal cylic import dependency. Users should update imports of the :class:`~cocotb.task.Task` to import from the top-level ``cocotb`` namespace. (:pr:`3067`)
```
Comment (marlonjames) [inline documentation/source/release_notes.rst:44]:
```suggestion
- Moved :class:`cocotb.task.Task` and friends to ``cocotb.task`` module to alleviate internal cyclic import dependency. Users should update imports of the :class:`~cocotb.task.Task` to import from the top-level ``cocotb`` namespace. (:pr:`3067`)
```
Noticed this in the 1.8 release notes.

File: documentation/source/newsfragments/README.rst
Comment (cmarqu) [inline documentation/source/newsfragments/README.rst:1]: This file was deleted accidentally I guess?

File: documentation/source/release_notes.rst (hunk 1 above)
Comment (imphil) [reply documentation/source/release_notes.rst]: It's minor enough that I'll keep things as-is in stable/1.8, but I took the change for master. Thanks!

File: documentation/source/newsfragments/README.rst
Comment (imphil) [reply documentation/source/newsfragments/README.rst:1]: Yes, that was unintended. Fixed.

**Review by ktbarrett — APPROVED** [review]

**Pull request merged**